package.json

```diff
@@ -15,19 +17,29 @@
   },
   "files": [
     "build",
-    "app.plugin.js"
+    "app.plugin.js",
+    "expo-module.config.json",
+    "ios"
   ],
   "keywords": [
     "react-native",
     "expo",
-    "hypertrack-sdk-react-native"
+    "hypertrack-sdk-react-native",
+    "HyperTrack",
+    "HyperTrackSDK"
   ],
   "repository": "https://github.com/hypertrack/sdk-expo",
   "author": "HyperTrack <help@hypertrack.com> (https://hypertrack.com)",
   "devDependencies": {
-    "expo-module-scripts": "^2.0.0"
+    "expo": "^46.0.13",
+    "@expo/config-plugins": "^4.1.5",
+    "expo-module-scripts": "^2.0.0",
+    "expo-modules-core": "^0.11.4"
   },
-  "dependencies": {
-    "@expo/config-plugins": "^4.1.5"
+  "dependencies": { },
+  "peerDependencies": {
+    "expo": "*",
+    "react": "*",
+    "react-native": "*"
```

Comment:
i saw this somewhere and thought it's a good idea to include it, but maybe it's not needed @poterstar ?

Reply:
this plugin doesn't depend on react or react-native it depends on the expo, and the expo itself depends on react and react-native.

So we can remove react, react-native from here and move expo from devDependecies to peerDependecies:

```
  "peerDependencies": {
    "expo": "^46.0.13"
  },
```

Reply:
probably we should also add `hypertrack-sdk-react-native` here?